components/script/dom/htmliframeelement.rs

```diff
@@ -544,6 +544,11 @@ impl HTMLIFrameElementMethods for HTMLIFrameElement {
     // https://html.spec.whatwg.org/multipage/#dom-dim-height
     make_dimension_setter!(SetHeight, "height");
 
+    // https://html.spec.whatwg.org/multipage/obsolete.html#other-elements,-attributes-and-apis:attr-iframe-frameborder
+    make_getter!(FrameBorder, "frameborder");
+    // https://html.spec.whatwg.org/multipage/obsolete.html#other-elements,-attributes-and-apis:attr-iframe-frameborder
```

Comment:
I think the multipage/foo.html links are discouraged, and multipage/  only is preferred.